contrib/inventory/azure_rm.py

```diff
@@ -311,10 +313,18 @@ def __init__(self, args):
             tenant = self.credentials.get('tenant')
             if not tenant:
                 tenant = 'common'
-            self.azure_credentials = UserPassCredentials(self.credentials['ad_user'],
-                                                         self.credentials['password'],
-                                                         tenant=tenant,
-                                                         cloud_environment=self._cloud_environment)
+
+            # # Default to Xplat Client ID
+            LEGACY_CLIENT_ID = '04b07795-8ddb-461a-bbee-02f9e1bf7b46'
```

Comment:
put az cli client id in ansible code raises some concern since it's not right thing to do.